commodore/cluster.py

```diff
@@ -120,47 +120,71 @@ def read_cluster_and_tenant() -> Tuple[str, str]:
 
 
 def render_target(
-    inv: Inventory, target: str, components: Iterable[str], bootstrap=False
+    inv: Inventory,
+    target: str,
+    components: Iterable[str],
+    bootstrap: bool = False,
+    # pylint: disable=unsubscriptable-object
+    component: Optional[str] = None,
 ):
-    if not bootstrap and target not in components:
+    if not component:
+        component = target
+    if not bootstrap and component not in components:
         raise click.ClickException(f"Target {target} is not a component")
 
     classes = [f"params.{BOOTSTRAP_TARGET}"]
-    parameters = {}
+    parameters: Dict[str, Union[Dict, str]] = {
+        "_instance": target,
+    }
 
-    for component in components:
-        defaults_file = inv.defaults_dir / f"{component}.yml"
+    for c in components:
+        defaults_file = inv.defaults_dir / f"{c}.yml"
```

Comment:
Oh, I must have missed one.

```suggestion
        defaults_file = inv.defaults_file(c)
```

Reply:
maybe I messed that one up when rebasing...